packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -399,7 +397,7 @@ export const validateCart = async (
     offerToOrderItemInput
   )
 
-  if (changes.length === 0) {
+  if (changes.length === 0 && orderForm.orderFormId === orderNumberFromCart) {
```

Comment:
This condition is already addressed earlier

```suggestion
  if (changes.length === 0) {
```

Reply:
good catch!